samples/cpp/image_generation/stable_diffusion_export_import.cpp

```diff
@@ -0,0 +1,121 @@
+// Copyright (C) 2023-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "imwrite.hpp"
+#include "openvino/genai/image_generation/text2image_pipeline.hpp"
+#include "progress_bar.hpp"
+
+int32_t main(int32_t argc, char* argv[]) try {
+    OPENVINO_ASSERT(argc == 3, "Usage: ", argv[0], " <MODEL_DIR> '<PROMPT>'");
+
+    const std::string models_path = argv[1], prompt = argv[2];
+
+    std::filesystem::path root_dir = models_path;
+
+    // export/import with a pipeline.export_model() method
+    {
```

Comment:
Move to separate functions. That way you call also include scoped comments into the names

Reply:
Done